src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,100 @@
+package duke;
+
+
+
+import javafx.application.Application;
+import javafx.stage.Stage;
+
+import java.util.Scanner;
+
+/**
+ * Duke is a task management application that allows users to manage their tasks.
+ * It provides a command-line interface and a JavaFX GUI for users to interact with their task list.
+ */
+public class Duke extends Application {
```

Comment:
Perhaps consider giving your chatbot a unique name and personality? Would be fun to play around 😆